Tests/test_MarkovModel.py

```diff
@@ -20,6 +23,11 @@
     warnings.simplefilter("ignore", UserWarning)
     from Bio import MarkovModel
 
+try:
```

Comment:
Can you use just ``from Bio._py3k import StringIO`` here?

It is shorter, and will make it slightly easier to remove all the Python 2 specific code one day.